packages/sanes-chrome-extension/src/extension/background/actions/createPersona/requestHandler/index.unit.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Request, RequestHandler } from './index';
+
+describe('RequestHandler', () => {
+  const REQUEST_ONE: Request = {
+    id: 1999,
+    type: 'getIdentities',
+    reason: 'Reason 1',
+    sender: 'Sender 1',
+    accept: () => {},
+    reject: () => {},
+  };
+
+  const REQUEST_TWO: Request = {
+    id: 2999,
+    type: 'signAndPost',
+    reason: 'Reason 2',
+    sender: 'Sender 2',
+    accept: () => {},
+    reject: () => {},
+  };
+
+  const REQUEST_THREE: Request = {
+    id: 3999,
+    type: 'getIdentities',
+    reason: 'Reason 3',
+    sender: 'Sender 3',
+    accept: () => {},
+    reject: () => {},
+  };
+
+  beforeEach(() => {
+    RequestHandler.load();
+    RequestHandler.add(REQUEST_ONE);
+    RequestHandler.add(REQUEST_TWO);
+    RequestHandler.add(REQUEST_THREE);
+  });
+
+  it('re-initializes itself', () => {
+    RequestHandler.load();
+
+    expect(RequestHandler['instance'].length).toBe(0);
```

Comment:
As I said in the review of the other PR, no more violating private accessors 👴 . You can use `RequestHandler.requests(). length`